frontend/assets/sass/main.scss

```diff
@@ -9,22 +9,26 @@
 @import "abstracts/placeholders";
 @import "abstracts/vars.color";
 @import "abstracts/vars.type";
-@import "abstracts/vars.uikit";
 
 
-// 2. Vendor & plugin styles.
-@import "uikit/src/scss/variables-theme.scss";
-@import "uikit/src/scss/mixins-theme.scss";
-@import "uikit/src/scss/uikit-theme.scss";
+// 2. UIKit Core & Theme.
+@import "theme/variables"; // needs to come before variables-theme
+@import "uikit/src/scss/variables-theme";
```

Comment:
No, it seems the general use case for overriding variables define with `!default` is to place them before. (see: https://robots.thoughtbot.com/sass-default)

This order is also what is suggested in the uikit docs in this section: https://getuikit.com/docs/sass#how-to-structure-your-theme